acceptance-tests/src/test/java/io/blueocean/ath/offline/BitbucketServerTest.java

```diff
@@ -0,0 +1,66 @@
+package io.blueocean.ath.offline;
+
+import com.cdancy.bitbucket.rest.BitbucketClient;
+import com.cdancy.bitbucket.rest.options.CreateRepository;
+import com.google.inject.Inject;
+import io.blueocean.ath.ATHJUnitRunner;
+import io.blueocean.ath.Login;
+import io.blueocean.ath.WaitUtil;
+import io.blueocean.ath.pages.blue.DashboardPage;
+import org.apache.log4j.Logger;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.io.IOException;
+
+@Login
+@RunWith(ATHJUnitRunner.class)
+public class BitbucketServerTest {
+
+    private static Logger LOGGER = Logger.getLogger(BitbucketServerTest.class);
+
+    @Inject
+    WaitUtil wait;
+
+    @Inject
+    DashboardPage dashboardPage;
+
+    @Test
+    public void testJenkinsfileCreate() {
+        BitbucketClient client = BitbucketClient.builder()
+            .endPoint("http://127.0.0.1:7990/bitbucket")
```

Comment:
Configurable via system properties or some other method? We should probably find a way to put a bunch of properties that are easily injected into the app, some config file then have a `SystemProperties.lookup` sort of thing that checks a few places and allows overrides down to the actual `System.getProperty` level

Reply:
maybe i'll just add that to my PR. it would be useful for me to just set it once and forget it

Reply:
not actually sure how to make this work on CI myself - the selenium one works as it uses the --net param to docker, but I dont' think we can do that for this. The IP would be different, but not sure what it would be.